pages/common/vala.md

```diff
@@ -0,0 +1,17 @@
+# vala
+
+> Vala code runner.
+> Tutorial: <https://wiki.gnome.org/Projects/Vala/Tutorial>.
+> Doc homepage: <https://valadoc.org/>.
```

Comment:
I'm not sure we support having more than 1 link as it needs to be prefaced by `More information`. Perhaps we could just change this to the following?

```suggestion
> More information: <https://wiki.gnome.org/Projects/Vala>.
```